hack/build-release.sh

```diff
@@ -6,7 +6,18 @@ source ./hack/version-util.sh
 
 mkdir -p tmp/
 
-ytt -f config/ -f config-release -v secretgen_controller_version="$(get_sgctrl_ver)" | kbld -f- > ./tmp/release.yml
+export version="$(get_sgctrl_ver)"
+
+yq eval '.metadata.annotations."secretgen-controller.carvel.dev/version" = env(version)' -i "config/package-bundle/contents/deployment.yml"
```

Comment:
note to future selves: this will make git directory dirty and kbld will record that.